app.js

```diff
@@ -0,0 +1,357 @@
+    optionMobile: ['CANOPY', 'Back End Dev', '2015'],
+    description:
+      " Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featuredImage: './images/photo4dt.png',
+    desktopImage: './images/photo4dt.png',
+    technologies: ['html', 'css', 'javascript', 'git hub', 'ruby', 'Bootstrap'],
+    linktoliveversion: 'https://sonyco-4u.github.io/Portfolio-microverse2/',
+    linktosource: 'https://github.com/sonyco-4u/Portfolio-microverse2',
+  },
+];
+
+function grab(e) {
+  return document.getElementById(e);
+}
+
+const Openpopup = ({
+  name,
+  description,
+  featuredImage,
+  desktopImage,
+  technologies,
+  linktoliveversion,
+  linktosource,
+  option,
+  optionMobile,
+}) => () => {
+  let displayModal = '';
+  modal.classList.add('show-modal');
+
+  displayModal += `
+         <div class="modal-head">
+          <h3 class="modal-title">${name}</h3>
+         </div>
+         <ul class="card-detail-header-container">
+             <li class="card-detail canopy-modal">${width > 1024 ? option[0] : optionMobile[0]
+}</li>
+             <li class="card-detail canopy-modal option-modal"><span class="circle-desktop"></span> ${width > 1024 ? option[1] : optionMobile[1]
+}</li>
+             <li class="card-detail canopy-modal option-modal"><span class="circle-desktop"></span>${width > 1024 ? option[2] : optionMobile[2]
+} </li>
+           </ul>
+            
+          <div class='modal-img-container' >
+            <img src='${width > 1024 ? desktopImage : featuredImage
+}' alt="modal picture" class="modal-img">
+          </div>
+          <div class="modal-text">
+            <div class="popup-description">
+            ${description}
+            </div>
+            <div>
+              <ul class="anchor-div technologies-langu">
+                <li><a href="#">${technologies[0]}</a></li>
+                <li><a href="#">${technologies[1] ? technologies[1] : ''
+}</a></li>
+                <li><a href="#">${technologies[2] ? technologies[2] : ''
+}</a></li>
+                <li><a href="#">${technologies[3] ? technologies[3] : ''
+}</a></li>
+              </ul>
+              <div class="modal-buttons ">
+                <a href='${linktoliveversion}' class="source-btn mr source-desktop">
+                  See Live
+                  <img src="images/Live-icon.svg" class="iconlink" alt="live icon">
+                </a>
+                <a href='${linktosource}' class="source-btn mr">
+                  See Source
+                <img src="images/Github-icon.svg" class="iconlink" alt="Git hub">
+                </a>
+              </div>
+            </div>
+          </div>
+      `;
+
```

Comment:
- [ ] Kindly make sure your popup design looks like the Figma design in both the mobile and desktop view by respecting the image sizes layout...

Your design | The template
------|------
![image](https://user-images.githubusercontent.com/65068771/190529425-5946e00e-fb8e-44cd-8b09-b8f1e3ba8d26.png) | ![image](https://user-images.githubusercontent.com/65068771/190529354-545e65db-4a34-4677-977b-2aa9bbac0fa4.png)
![image](https://user-images.githubusercontent.com/65068771/190529601-d5d36ce7-14cb-49a7-b53e-94a441f4ca78.png) | ![image](https://user-images.githubusercontent.com/65068771/190529544-df8ca96f-4178-4de7-abbb-028e18fe4803.png)

Reply:
Thank you for the corrections , I made all the necessary changes.